<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeWgt</class>
 <widget class="QWidget" name="homeWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>595</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="contrlWgt" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#contrlWgt{
background:#2a2b2b;
border:1px solid #000000;
border-left:none;
border-right:none;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="consoleTabWidget" name="tabConsoleWgt_" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="supTabWidget" name="tabPlayWgt_" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="supTabWidget" name="tabReplayWgt_" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="supTabWidget" name="tabDeviceWgt_" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="supTabWidget" name="tabStorageWgt_" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="supTabWidget" name="tabVRMapWgt_" native="true">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="supTabWidget" name="tabVRViewWgt_" native="true">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="supTabWidget" name="tabIOTPlatformWgt_" native="true">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="consoleWgt" name="consoleWgt_"/>
        <widget class="deviceManageWgt" name="deviceWgt_"/>
        <widget class="playWidget" name="playWgt_"/>
        <widget class="replayWgt" name="replayWgt_"/>
        <widget class="storageWgt" name="storageWgt_"/>
        <widget class="VRMapWidget" name="VRMapWgt_"/>
        <widget class="VRViewWidget" name="VRViewWgt_"/>
        <widget class="IOTWgt" name="IOTPlatWgt_"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>consoleWgt</class>
   <extends>QWidget</extends>
   <header>consolewgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>consoleTabWidget</class>
   <extends>QWidget</extends>
   <header>consoletabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>supTabWidget</class>
   <extends>QWidget</extends>
   <header>suptabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>replayWgt</class>
   <extends>QWidget</extends>
   <header>replaywgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>storageWgt</class>
   <extends>QWidget</extends>
   <header>storagewgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>deviceManageWgt</class>
   <extends>QWidget</extends>
   <header>devicemanagewgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>playWidget</class>
   <extends>QWidget</extends>
   <header>playwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VRMapWidget</class>
   <extends>QWidget</extends>
   <header>VRMap/VRMapWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VRViewWidget</class>
   <extends>QWidget</extends>
   <header>VRView/VRViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IOTWgt</class>
   <extends>QWidget</extends>
   <header>iotwgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
